aws/application/application.template

```diff
@@ -62,12 +62,6 @@ Parameters:
   pPostMvpEnabled:
     Type: String
     Default: ''
-  pProsecutionConcludedListenerEnabled:
-    Type: String
-    Default: ''
-  pProsecutionConcludedScheduleEnabled:
-    Type: String
-    Default: ''
   pCreateLinkCpStatusJobQueue:
     Type: String
     Default: ''
```

Comment:
Missed reference to `pLaaProsecutionConcludedQueue` environment variable on lines 68 - 70.